src/kde_analysis/eval_m1m2chieff_poiss_weights.py

```diff
@@ -206,79 +469,138 @@ def get_rate_m_chieff2D(m1_query, m2_query, Rate):
     m1 = samples[:, 0]  # First column corresponds to m1
     m2 = samples[:, 1]  # Second column corresponds to m2
     cf = samples[:, 2]
-    samples2 = np.vstack((m2, m1, cf)).T
-    # Combine both samples into one array
-    symmetric_samples = np.vstack((samples, samples2))
 
     # Determine weights based on vt_weights flag
     if boots_weighted:
         if vt_weights:
-            weights_over_VT = poisson_weights / vt_vals
-            # Duplicate weights for symmetric samples (m1 <-> m2)
-            weights = np.tile(weights_over_VT, 2)
+            weights_over_VT = weights / vt_vals
+            weights = weights_over_VT
         else:
-            weights = np.tile(poisson_weights, 2)
+            weights = poisson_weights
```

Comment:
this will be an error as we don't have an object `poisson weights` any more .. since there is nothing to do here we just `pass`